External/Plugins/HaXeContext/Context.cs

```diff
@@ -1615,4 +1673,48 @@ public HaxeCompletionCache(ASContext context, MemberList elements, MemberList ot
             OtherElements = otherElements;
         }
     }
+
+    /// <summary>
+    /// Represents a semantic version, see http://semver.org/
+    /// </summary>
+    class SemVer
```

Comment:
This class is potentially useful outside of `Context.cs` or even the `HaXeContext` plugin, but I'm not sure where else to put it. `FlashDevelop.Utilities` maybe?